67_RayQueryGeometry/main.cpp

```diff
@@ -0,0 +1,1010 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#include "common.hpp"
+
+struct SCameraParameters
+{
+	float camPos[3];
+
+	float MVP[4 * 4];
+	float V[3 * 4];
+	float P[3 * 4];
+};
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/157/files#r1829166059

Reply:
still a TODO